### PR TITLE
Parks Local

### PR DESCRIPTION
**Pull request by miriamseely**
Current metadata draft for Utah Parks Local. There was very little available information on this dataset, so definitely let me know if there are important details missing. Thank you!

### CODE REVIEW

File: metadata/recreation/parks_local.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+This dataset has been made available to the public for general analytic and cartographic purposes.
+
+### What does the dataset represent?
+
+Each polygon represents the approximate geographic extent of a given park and contains attributes such as the name of the park, type (neighborhood park, soccer field, equestrian, etc.), and area in acres.
+
+### How was the dataset created?
+
+UGRC created this dataset in collaboration with county government partners. County officials sent us a list of parks and public spaces to be included, and we digitized these places as polygons on top of [NAIP imagery](https://naip-usdaonline.hub.arcgis.com/). This dataset is updated as counties submit new parks to be included or existing parks to be removed.
```
Comment (ZachBeck) [inline metadata/recreation/parks_local.md:33]: Maybe something like... parks were originally digitized using NAIP imagery, followed by higher resolution sources as they became available.

**Review by ZachBeck — COMMENTED** [review]
Looks good.

**Comment by steveoh** [comment]
@ZachBeck are you the data creator/maintainer for parks? Implying we don't have external stewards providing updates?

**Comment by ZachBeck** [comment]
I consider myself neither of those but I've added a couple parks here and there over the years. The bulk of the work was done by Gordon and Cindy, there might have been a few counties that sent shapefiles of their parks to get them started. Unfortunately there are no external stewards that I'm aware of.

**Comment by gregbunce** [comment]
This could be another OSM derived dataset

**Comment by steveoh** [comment]
> This could be another OSM derived dataset

Yeah something should happen for a dataset without stewards. Is there a running list for OSM candidates?

**Comment by miriamseely** [comment]
@gregbunce @steveoh @ZachBeck I'm looking to merge this layer but didn't want to close the pull request if the discussion about OSM candidates still needs to be had. Is this something we need to figure out before checking this layer off, or can I go ahead and merge it? Perhaps we should have a broader conversation about stewardless layers? I have a few more coming up that don't seem to have assigned stewards

**Comment by gregbunce** [comment]
i would say go ahead and move forward with this PR (merge it,). 

I will create and share a list of layers that are good candidates for OSM and my suggestions for what that might look like, but we don't need to have that conversation here. 

carry on! thanks Miriam!

**Pull request merged**